technical_masterplan.md

```diff
@@ -218,6 +218,49 @@ This component vocabulary section aims to briefly define the components involved
 - **Dialog Broker (DB)** - is responsible for providing the fulfillable intents available for a resolved VRS record (e.g. where resolved VRS record "BigGrocery", it’s fulfillable intents might be "order product, check order status, add to shopping list". These fulfillable intents can execute remotely on the DM or download locally on the device.
 - **Dialog Manager (DM)** - handles the dynamic response of the conversation. It provides a more personalized response based on the action provided by the NLP to send back to the user.
 
+## Why are new and additional voice standards needed?
+
+Broadly stated, voice has evolved over the past decade from call center-centric recognition to a consumer-owned, platform-enabled all-purpose answer machine. In its next phase – one that is unfolding now – it will become the enabler of contextually-aware dialogue and true personalized assistance.
+The next step in the voice standards journey is address the transitions noted above, as well as these more recent evolutions: • Rapid advances in the AI-enabled voice ecosystem and value chain. A unique industry is now being created with independent voice assistance by world-leading technology firms. Voice is increasingly the entry point of a data-consuming engine of technology and economic growth. Today’s voice questions are different, and much larger than before.
+
+• Ongoing advances in AI-enabled voice technology. Several of the standards listed below were developed to enable interactive voice response systems for telephone call centers. We now stand at the edge of contextually-aware voice functionality that enables human-like dialogue.
+
+• The emergence of platform-independent voice assistants. This calls into question platform and assistant interoperability, in which an enterprise-owned independent assistant can speak with all others, regardless of platform or channel. It suggests a standardization of components, the building blocks of voice assistants – components that have emerged from many of the standards listed below.
+
+As the OVN begins its work, it does so with the desire to learn from and build upon a significant number of widely-adopted and currently-relevant standards for voice assistance. The list below identifies existing voice standards that may guide the functionality and definition of voice assistant components (Dialogue Manager, TTS, ASR, NLU, and many others) and governance protocols (a potential destination registry) now in development by the OVN Technical Committee. It is acknowledged that this list may not be comprehensive.
```

Comment:
I think a lot of the content here is already explained above in other introductory paragraphs. Rather than add a whole new section, perhaps take this content and see if it can fit in other sections we already have above, and if it doesn't fit, maybe what's left deserves a new section. Either way, if you want to keep it all intact, moving it up high to the introductory areas is probably best.